flask/ctx.py

```diff
@@ -133,20 +135,25 @@ def copy_current_request_context(f):
         def index():
             @copy_current_request_context
             def do_some_work():
-                # do some work here, it can access flask.request like you
-                # would otherwise in the view function.
+                # do some work here, it can access flask.request or
+                # flask.session like you would otherwise in the view function.
                 ...
             gevent.spawn(do_some_work)
             return 'Regular response'
 
     .. versionadded:: 0.10
+
+    .. versionchanged:: 1.0.3
```

Comment:
This feels like a 1.1 change instead of a bugfix change. And since you branched off master, that's what would happen anyway.